src/endpoints/AliasesEndpoint.js

```diff
@@ -61,13 +61,14 @@ import Alias from "../models/Alias.js";
 	 * Update an existing alias by its name.
 	 *
 	 * Raises an error if the alias does not exist or belong to the currently logged in account.
-	 * @param {Alias} alias the alias to update.
-	 * @param {string} [newName] optional updated name of the alias. if not included alias retains the same name.
-	 * @param {string} [newPayload] optional updated payload of the alias. if not included alias retains the same payload.
+	 * @param {string} aliasName the alias to update.
+	 * @param {Object} updates
+	 * @param {string} [updates.newName] optional updated name of the alias. if not included alias retains the same name.
+	 * @param {string} [updates.newPayload] optional updated payload of the alias. if not included alias retains the same payload.
 	 * @returns {Promise<Alias>}
 	 */
-	async updateAlias(alias, newName = alias.name, newPayload = alias.payload) {
-		let updatedAliasDTO = await this.request(`aliases/${alias.name}`, {
+	async updateAlias(aliasName, {newName, newPayload}) {
```

Comment:
@farnazj this updateAlias method changed. Previously it took an `Alias` model and set the `newName` and `newPayload` to the `name` and `payload` values of the `Alias` model if the user did not pass those in.

I want this method to match the rest of our API and allow users to call `updateAlias` with just the `aliasName`.

Could we make sure that if the user doesn't pass the `name` or `payload` the update only changes the fields they passed in.